samples/web/content/manual-test/iframe-apprtc/index.html

```diff
@@ -15,6 +15,6 @@
 </head>
 <body>
   AppRTC in an &lt;iframe&gt; element:<br>
-  <iframe width="90%" height="90%" src="http://apprtc.appspot.com"></iframe>
+  <iframe width="90%" height="1200px" src="http://apprtc.appspot.com"></iframe>
```

Comment:
Hmm... That's odd (see [MDN](https://developer.mozilla.org/en/docs/Web/HTML/Element/iframe), etc.) 

Oh well – maybe percentage width with absolute height? 

Will merge.